README.md

```diff
@@ -183,7 +183,7 @@ If you do end up modifying the migrations for the database, you will need to rer
 7. Run tests:
 
 ```bash
-go test ./...
+go test --tags test ./…
```

Comment:
Does double dash work? I've never used that format :shrug: 
```suggestion
go test -tags test ./…
```

Reply:
I use it by habit, but I've been trying to switch to single dash since it's less typing!